### PR TITLE
dsc-13

### PR DESCRIPTION
**Pull request by katrze89**
…t and md files

### CODE REVIEW

File: dsc/templates/dsc/inc/deviceserver_documentation.html [hunk 1]
```diff
@@ -46,16 +47,14 @@ <h4>{{doc.get_documentation_type_display}}</h4>
 {% if readme or readme_link %}
     <div style="word-wrap: break-word;">
         <h4>README</h4>
-        {% if rst %}
-            {{ readme|apply_markup:"restructuredtext" }}
-        { % elif md %}
-            {{readme|apply_markup:"markdown"}}
+        {% if rst or deviceserver.description|find_match:'.*:.*:.*'%}
```
Comment (piogor) [inline dsc/templates/dsc/inc/deviceserver_documentation.html]: Should it be test on readme?:
```suggestion
        {% if rst or readme|find_match:'.*:.*:.*'%}
```

**Review by piogor — CHANGES_REQUESTED** [review]
Thanks, @katrze89 !
There is one suggestion for change :-)

**Comment by katrze89** [comment]
@piogor Of course. I'm sorry for overlooking.

**Review by piogor — APPROVED** [review]
Thanks, @katrze89!

**Pull request merged**